src/apiflask/commands.py

```diff
@@ -40,7 +38,7 @@ def spec_command(format, output, indent):
     json_indent = None if indent == 0 else indent
 
     if spec_format == 'json':
-        spec = json.dumps(spec, indent=json_indent)
+        spec = current_app.json.dumps(spec, indent=json_indent)
```

Comment:
This change wasn't necessary to get the `flask spec` command working, but it seemed like it might be helpful, so I included it as well.